karols-frontend/src/components/CardModel.js

```diff
@@ -105,7 +106,8 @@ const CardModel = ({
                   transform: "translateY(calc(0px + 1em))"
                 }}
               >
-                {description}{" "}
+                {description} <br />
```

Comment:
must appear in calendarContainer
si vous ne trouver pas de créneaux appelez au...